File: pytools/examples/php/server/protocol/AckSceneExit.php
<?php
namespace protocol;

use network\Packet;


class AckSceneExit
{
	private $uid = 0;


	public function encode()
	{
		$packet = new Packet();
		$packet->writeU32($this->uid);
		return $packet->encode(Msg::$P_ACK_SCENE_EXIT);
	}


	public function setUid($uid)
	{
		$this->uid = $uid;
	}

}

File: pytools/examples/java/server/protocol/AckSceneExit.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class AckSceneExit
{
	private int uid;


	public byte[] encode()
	{
		Packet packet = new Packet();
		packet.writeInt(uid);
		return packet.encode((short)Msg.P_ACK_SCENE_EXIT);
	}



	public void setUid(int uid)
	{
		this.uid = uid;
	}

}
